covalent_ui/app.py

```diff
@@ -105,7 +107,7 @@ def get_home(request: Request, rest_of_path: str):
         "app:fastapi_app",
         host=host,
         port=port,
-        debug=DEBUG,
+        debug=log_to_file,
```

Comment:
@Prasy12 is there a reason why we're enabling debug on uvicorn if we enable sdk logging?

Reply:
Fixed